### PR TITLE
Change Poller time validation

### PR DESCRIPTION
**Pull request by joseUPV**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by joseUPV** [comment]
We have 10 minutes step, validate.php always fails in last 5 minutes.

File: LibreNMS/Validations/Poller.php [hunk 1]
```diff
@@ -80,10 +80,11 @@ private function checkDuplicatePollerEntries(Validator $validator)
 
     private function checkLastPolled(Validator $validator)
     {
+        $period = (int)Config::get('rrd.step', 300);
```
[the remark this answers is not in the document]
Comment (joseUPV) [reply LibreNMS/Validations/Poller.php]: But the 300 is also in:
- function checkDeviceLastPolled
- function checkDevicePollDuration

This is the same line that is in function checkDevicePollDuration (line 130, 129 in the original code.).

[the remark this answers is not in the document]
Comment (Jellyfrog) [reply LibreNMS/Validations/Poller.php]: Doesn't really change the fact

[the remark this answers is not in the document]
Comment (joseUPV) [reply LibreNMS/Validations/Poller.php]: Isn't 300 default value?
If there is no config value there will be problems, in original code SQL has 5 minutes fixed.

[the remark this answers is not in the document]
Comment (joseUPV) [reply LibreNMS/Validations/Poller.php]: Isn't 300 default value?
If there is no config value there will be problems, in original code SQL has 5 minutes fixed.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-61-release-changelog-february-2020/11145/1

**Pull request merged**